ntc_templates/templates/cisco_ios_show_platform.textfsm

```diff
@@ -0,0 +1,12 @@
+Value INDEX (\d+)
+Value PORTS (\d+)
+Value MODEL (\S+)
+Value SERIAL (\S+)
+Value MAC (\S+)
+Value HW_VER (\S+)
+Value SW_VER (\d+\.\d+\.\d+)
+
+Start
+  ^Switch.*
+  ^--.*
+  ^\s${INDEX}\s+${PORTS}\s+${MODEL}\s+${SERIAL}\s+${MAC}\s+${HW_VER}\s+${SW_VER}
```

Comment:
```suggestion
  ^\s*${INDEX}\s+${PORTS}\s+${MODEL}\s+${SERIAL}\s+${MAC}\s+${HARDWARE_VERSION}\s+${SOFTWARE_VERSION}\s*
  ^\s*$$
  ^. -> Error
```
We require all new templates to Error on unidentified lines to help ensure we are capturing what we expect to capture.